src/pages/Version.tsx

```diff
@@ -48,39 +50,63 @@ export const VersionPage: React.FC = () => {
   const [redirectURL, setRedirectURL] = useState(undefined);
   const [isLoadingData, setIsLoadingData] = useState(true);
 
-  const [getPatch, { data: patchData }] = useLazyQuery<
+  const { error: isPatchOrVersionError } = useQuery<
```

Comment:
nit: can this error be called something more related to the new query name

Reply:
Nice catch!